src/components/line-chart-audification/line-chart-audification.component.spec.ts

```diff
@@ -21,26 +22,31 @@ describe('LineChartAudificationComponent', () => {
   beforeEach(() => {
     TestBed.configureTestingModule({
       imports: [
-        DataModule,
+        MatCardModule,
       ],
       declarations: [
         LineChartComponent,
       ],
     });
     const hostFixture = TestBed.createComponent(LineChartComponent);
     const host = hostFixture.componentInstance;
-    host.measureName = 'activeUsers';
-    host.ngOnChanges({ measureName: new SimpleChange(null, host.measureName, true) });
+    const time = new Date();
+    host.meta = createLineChartMeta(
+      'Title',
+      () => [{
+        label: '',
+        points: [{ x: time, y: 0 }, { x: new Date(time.getTime() + 10), y: 1 }],
+      }],
+    );
 
     TestBed.resetTestingModule();
     TestBed.configureTestingModule({
       imports: [
```

Comment:
Apologies if this is just me not understanding TestBed (we use a slightly different wrapper around TestBed), but most of our tests import the module under test, which would include the dependencies like ScreenReaderModule or MatCardModule. So our setup for this would be 
```
imports: [
  LineChartAudificationModule,
],
providers: [...]
```
And we would not declare the component. Is this not the recommended pattern for TestBed setup?

Reply:
That sounds better, so we don't need to re-specify the dependencies for testing. I'll work on it to see if it works.

Reply:
I think both are reasonable. These tests started from the default `spec.ts` generated by Angular CLI, and it declares the components in TestBed. Besides, the these test files are named `xxx.component.spec.ts`, which stand for tests for `xxx.component.ts`, so it is reasonable to declare the component directly.
I think a better way is to move the test to module level, like renaming it to `xxx.module.spec.test`

Reply:
Moved to #54 . Will be resolved in separate PRs.